### ISSUE TITLE
UnicodeDecodeError in task, Generate p12 export password

### ISSUE THREAD

**Issue by azdkj532** [trailofbits/algo#1573]
I got task failed with error message `"UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd3 in position 0: invalid continuation byte"`, so I dive into roles trying to find where bug is.
The problem is this python script in [`Generate p12 export password` ](https://github.com/azdkj532/algo/blob/master/roles/common/tasks/facts.yml#L10) : 
```
import sys,string;
chars=string.ascii_letters + string.digits + "_@"; 
print("".join([chars[ord(c) % 64] for c in list(sys.stdin.read())]))
```

In python3, default encoding method is utf8, and random byte can not be decoded to valid unicode string.
Maybe we can use another way to generate password.

**Comment by davidemyers** [comment]
The current version of Algo requires Python 2. Python 3 will be supported soon and you can try it out by using the branch in PR #1549 where this issue has already been addressed.

**Comment by azdkj532** [comment]
@davidemyers I have a quick fix of this. Should I send PR ?
https://github.com/azdkj532/algo/commit/ec37deb19a2af6cd678be502ba2fbfee435ec951

**Comment by davidemyers** [comment]
I think there are other modules that will fail with Python 3 so I'm not sure if fixing this one issue is enough. Once the above PR is merged there will be full Python 3 support.

**Comment by jackivanov** [comment]
Fixed in #1574
